src/ui/shared/Search.tsx

```diff
@@ -1,67 +1,135 @@
-              />
-            </svg>
+            <XMarkIcon className="h-4 w-4" />
           </button>
         </div>
-      ) : (
-        <button
-          onClick={() => setIsOpen(true)}
-          className="flex items-center justify-center w-8 h-8 rounded-full bg-twd-background transition"
-        >
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            className="h-5 w-5 text-gray-300"
-            fill="none"
-            viewBox="0 0 24 24"
-            stroke="currentColor"
-            strokeWidth={2}
-          >
-            <circle cx="11" cy="11" r="8" stroke="currentColor" />
-            <line
-              x1="21"
-              y1="21"
-              x2="16.65"
-              y2="16.65"
-              stroke="currentColor"
-              strokeWidth={2}
-              strokeLinecap="round"
-            />
-          </svg>
-        </button>
       )}
     </div>
   );
 }
+
+
+
+
+//     <div className="relative flex items-center">
+//       {isOpen ? (
+//         <div className="flex items-center space-x-2 rounded-full bg-white px-4 py-2">
+//           <input
+//             type="text"
+//             placeholder="Search..."
+//             value={query}
+//             onChange={(e) => setQuery(e.target.value)}
+//             className="w-full bg-transparent focus:outline-none"
+//           />
+//           <button
+//             onClick={() => setIsOpen(false)}
+//             className="flex items-center justify-center"
+//           >
+//             <svg
+//               xmlns="http://www.w3.org/2000/svg"
+//               className="h-5 w-5 text-white"
+//               fill="none"
+//               viewBox="0 0 24 24"
+//               stroke="currentColor"
+//               strokeWidth={2}
+//             >
+//               <path
+//                 strokeLinecap="round"
+//                 strokeLinejoin="round"
+//                 d="M10 14l2-2m0 0l2-2m-2 2l-2-2m8 10a9 9 0 11-18 0 9 9 0 0118 0z"
+//               />
+//             </svg>
+//           </button>
+//         </div>
+//       ) : (
+//         <button
+//           onClick={() => setIsOpen(true)}
+//           className="flex items-center justify-center w-8 h-8 rounded-full bg-twd-background transition"
+//         >
+//           <svg
+//             xmlns="http://www.w3.org/2000/svg"
+//             className="h-5 w-5 text-gray-300"
+//             fill="none"
+//             viewBox="0 0 24 24"
+//             stroke="currentColor"
+//             strokeWidth={2}
+//           >
+//             <circle cx="11" cy="11" r="8" stroke="currentColor" />
+//             <line
+//               x1="21"
+//               y1="21"
+//               x2="16.65"
+//               y2="16.65"
+//               stroke="currentColor"
+//               strokeWidth={2}
+//               strokeLinecap="round"
+//             />
+//           </svg>
+//         </button>
+//       )}
+//     </div>
+//   );
+
```

Comment:
Possible to delete this? If so commit below
```suggestion
```

Reply:
deleted